msgpack.gemspec

```diff
@@ -12,10 +12,14 @@ Gem::Specification.new do |s|
   s.homepage = "http://msgpack.org/"
   s.require_paths = ["lib"]
   if /java/ =~ RUBY_PLATFORM
-    s.files = Dir['lib/**/*.rb', 'lib/**/*.jar']
+    s.files = Dir['lib/**/*.rb', 'lib/**/*.jar', 'LICENSE']
```

Comment:
Include the license document in the gem for the Java platform. This required so Rubygems can legally distribute the code.

https://github.com/msgpack/msgpack-ruby/blob/87ea58b7fb18439e7803fea292a1db4008af2e87/LICENSE#L95-L96